modules/openapi-generator-maven-plugin/pom.xml

```diff
@@ -64,6 +64,12 @@
             <version>${junit.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.apache.maven</groupId>
+            <artifactId>maven-resolver-provider</artifactId>
+            <version>3.9.4</version>
```

Comment:
can w use 3.9.6 instead which is the latest stable 3.x version?

Reply:
I've aligned it with other maven dependencies. Should I update other references to 3.9.6, also? Or just use a property to reference same maven version and update in another PR?

https://github.com/apache/maven/blob/master/pom.xml#L308

Reply:
Updated maven dependencies to 3.0.6 via same property. Now it is clearer that they belong together updated.